### DISCUSSION TITLE
1.x 升级指南

### DISCUSSION THREAD

**Discussion by ihaichao** [alibaba/formily#1779] [Ideas]
请问有没有 1.x 升级到 2.0 的指南、注意事项等文档

**Comment by janryWang** [comment]
https://v2.formilyjs.org/zh-CN/guide/upgrade

**Reply by ihaichao** [reply]
谢谢，另外想问下有没有 1.x 升级到 2.0 的性能数据对比

**Reply by janryWang** [reply]
https://zhuanlan.zhihu.com/p/364750033
